openadapt/visualize1.py

```diff
@@ -0,0 +1,209 @@
+from pprint import pformat
+
+from loguru import logger
+from nicegui import ui
+from tqdm import tqdm
+
+from openadapt import config
+from openadapt.crud import get_latest_recording
+from openadapt.events import get_events
+from openadapt.utils import (
+    EMPTY,
+    configure_logging,
+    display_event,
+    image2utf8,
+    row2dict,
+    rows2dicts,
+    plot_performance,
+)
+
+SCRUB = config.SCRUB_ENABLED
+if SCRUB:
```

Comment:
how about just `if config.SCRUB_ENABLED`?

Reply:
I declare SCRUB because it gets checked quite often in the main function.